gitCoreJGit/src/main/java/com/virtuslab/gitcore/impl/jgit/GitCoreLocalBranch.java

```diff
@@ -143,14 +181,18 @@ public String getBranchTypeString(boolean capitalized) {
 
     List<ReflogEntry> filteredReflogEntries = reflogEntryLists.flatMap(this::rejectExcludedEntries);
 
+    LOG.debug("Start walking through revlogs");
```

Comment:
or `reflogs`?

Reply:
yeah, given that `rev(ision) ~= commit` and `ref(erence) ~= branch` than these are more reflogs than revlogs

Reply:
`revlogs` is correct in my opinion coz this is log of revisions not references but @PawelLipski, can U speak out?

Reply:
Hence e.g. `git rev-parse` plumbing command - to `parse a given revision`, whether it's just a branch name or a more complex expression like `foo@{upstream}^2`

Reply:
but below in code we are walking thru commits not reference (aka branch) log

Reply:
even `RevCommit` var type indicate this

Reply:
Oh ok, right - but in such case I'd simply use `logs`, that's how git calls that (I've never heard of `rev(ision) logs` even though the name kinda makes sense)

Reply:
But generally, wrt. how stuff is named in git:

![image](https://user-images.githubusercontent.com/3383210/80148198-f76dd800-85b4-11ea-9bbf-40aabc77c9f4.png)

Reply:
Same for some CLI quirks - git's CLI is IMO one of the **worst** such popular in existence... or maybe rather it's so vast that it has some good spots (like `git grep`) and some terrible (`git checkout`, `git reset`, general inconsistency of how things are added/deleted etc.)